src/Symfony/Component/Process/Process.php

```diff
@@ -334,6 +335,10 @@ public function start(callable $callback = null/*, array $env = array()*/)
             $ptsWorkaround = fopen(__FILE__, 'r');
         }
 
+        if (!is_dir($this->cwd)) {
+            @trigger_error('The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.', E_USER_DEPRECATED);
```

Comment:
I think we should stick to my previous suggestion: on Windows, we should throw an exception instead of a deprecation, because on Windows, it's already broken, the getcwd fallback does not apply.

Reply:
Done